features/admin.claims.v1/components/edit/local-claim/edit-basic-details-local-claims.tsx

```diff
@@ -524,14 +599,65 @@ export const EditBasicDetailsLocalClaims: FunctionComponent<EditBasicDetailsLoca
             });
     };
 
+    const subAttributeDropdownOptions: DropDownItemInterface[] = useMemo(() => {
+        return fetchedAttributes
+            ?.filter((claim: Claim) => {
+                const isSystemClaim: boolean = claim.properties?.some(
+                    (property: Property) =>
+                        property.key === "isSystemClaim" && property.value === "true"
+                );
+                const isCurrentClaim: boolean = claim.claimURI === props.claim.claimURI;
+                const isAlreadySelected: boolean = subAttributes.includes(claim.claimURI);
+
+                return !isSystemClaim && !isCurrentClaim && !isAlreadySelected;
+            })
+            .map((claim: Claim) => ({
+                text: claim.claimURI,
+                value: claim.claimURI
+            }))
+            .sort(
+                (a: { text: string }, b: { text: string }) =>
+                    a.text.localeCompare(b.text)
+            );
+    }, [ fetchedAttributes, subAttributes, props.claim.claimURI ]);
```

Comment:
We already get the claim prop in L130 by destructuring props.

```suggestion
    }, [ fetchedAttributes, subAttributes, claim?.claimURI ]);
```

Reply:
Fixed by - https://github.com/wso2/identity-apps/pull/8111/commits/6f8d1b53919c0d00e80b9142da52b064cef38ebf